.solcover.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     'Migrations.sol',
     'mocks',
     'external',
+    'managers/lib/UintArrayUtilsLibrary.sol',
```

Comment:
What's the reason we're excluding this?

Reply:
Just having coverage issues. Not necessarily final. Experimenting.